src/step-3-logging/README.md

```diff
@@ -0,0 +1,5 @@
+# step-2
```

Comment:
```suggestion
# step-3
```

Why is this `step-3-logging` while the others have no other suffix?

Reply:
> Why is this step-3-logging while the others have no other suffix?

I don't think I understand this comment - the folders do all have suffixes?

Reply:
the other folders don't. `step-2` does not have any suffix, nor have the others in https://github.com/nearform/the-fastify-workshop/tree/master/src.

Why is this named `step-3-logging` instead of `step-3`?

Reply:
The link is to the old steps on master - we've added new steps and decided to put suffixes on them to make it easier to follow

Here's the new set of steps: https://github.com/nearform/the-fastify-workshop/tree/33e1d4aae47c6ec678a69bcdb7c4254992d84e71/src

Reply:
This readme is still wrong. This is step-3 not step-2.